terraform/cp4ba/terraform.tfvars.classic

```diff
@@ -0,0 +1,48 @@
+// Openshift cluster parameters
```

Comment:
You don't need this file for this example.  This was only in the `roks` module for provisioning a cluster.